.github/workflows/release.yml

```
@@ -14,7 +14,9 @@ jobs:
     permissions:
       contents: write
       id-token: write
+
     steps:
+      - name: Checkout Engine repo
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)